### DISCUSSION TITLE
New dictionary for network

### DISCUSSION THREAD

**Discussion by cidrblock** [streetsidesoftware/cspell-dicts#902] [Ideas]
Hey all-

I'm starting to find several network platform/protocol related words.

I didn't want to do a PR against an existing dictionary since a new dictionary/files might be cleaner, although I'm not sure I have the skills to add it.  I'm thinking 2 files, similar to how software-* is setup.

network-platforms
network-protocols

Examples:

Protocols:

snmp
bgp
ospf

Platforms/OS:

eos
ios
nxos

I thought I'd throw it out here for discussion, to see if the base configs could be set-up so I can add the words in a sane location

**Comment by Jason3S** [comment]
@cidrblock,

Thank you for the suggestion.

A single package can export multiple dictionaries. Since all of these are related, I think they can be put in software terms as separate dictionaries.

If you create a source file for each one, I can help with the next steps.

**Comment by cidrblock** [comment]
@Jason3S 

Thanks for the reply.  Here's a PR: 

https://github.com/streetsidesoftware/cspell-dicts/pull/919

I think those two files _should_ keep it pretty clean and organized, let me know if there is anything I can do to help
